zulip/integrations/bridge_with_slack/run-slack-bridge

```diff
@@ -6,10 +6,12 @@ import os
 import argparse
 import traceback
 import threading
+
+from typing import Any, Dict, Callable
```

Comment:
Is this a manual reorder or per linting? (there's no note in the commit text re this)
How did this pass CI linting in the first place? Is there a linter for this?
(isort in ZT wouldn't have the blank line above, which makes me query this too)

Reply:
I did the reordering manually. And the order has been that way for a long time, which means isort is not configured to run with this repo yet.

Reply:
I have reordered with isort and force pushed.